package-lock.json

```diff
@@ -5410,9 +5336,9 @@
       }
     },
     "node_modules/caniuse-lite": {
-      "version": "1.0.30001576",
-      "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001576.tgz",
-      "integrity": "sha512-ff5BdakGe2P3SQsMsiqmt1Lc8221NR1VzHj5jXN5vBny9A6fpze94HiVV/n7XRosOlsShJcvMv5mdnpjOGCEgg==",
+      "version": "1.0.30001565",
```

Comment:
BTW, with your change, you actually downgraded this dependency 1.0.30001576 -> 1.0.30001565

So, reverting this whole package-lock.json part is still needed

Reply:
Updated